src/current/_includes/cockroachcloud/download-the-binary.md

```diff
@@ -1,21 +1,29 @@
     <section class="filter-content" markdown="1" data-scope="mac">
-    If you have not done so already, run the first command in the dialog to install the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
+    If you have not done so already, run the first command in the dialog to install the latest downloadable version of the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach /usr/local/bin/
+    curl https://get.cockroachdb.com/dl/cockroach-latest.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-latest.darwin-10.9-amd64/cockroach /usr/local/bin/
     ~~~
 
+    {{site.data.alerts.callout_success}}
+    To download the ARM 64-bit architecture instead of the Intel 64-bit binary, replace all instances of `darwin-10.9-amd64` with `darwin-11.0.arm64` before running the command.
+    {{site.data.alerts.end}}
+
     </section>
 
     <section class="filter-content" markdown="1" data-scope="linux">
-    If you have not done so already, run the first command in the dialog to install the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
+    If you have not done so already, run the first command in the dialog to install the latest downloadable version of the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.linux-amd64.tgz | tar -xz && sudo cp -i cockroach-{{ page.release_info.version }}.linux-amd64/cockroach /usr/local/bin/
+    curl https://get.cockroachdb.com/dl/cockroach-latest.linux-amd64.tgz | tar -xz && sudo cp -i cockroach-latest.linux-amd64/cockroach /usr/local/bin/
```

Comment:
(same as previous note)
UI shows `cockroach-*/cockroach` instead of `cockroach-latest.linux-amd64/cockroach`.

Reply:
i.e. even though command is
`curl https://get.cockroachdb.com/dl/cockroach-latest.linux-amd64.tgz`
what actually gets downloaded is a versioned download, e.g.
- `cockroach-v23.1.14.darwin-10.9-amd64` or `cockroach-v23.1.14.linux-amd64`

so `sudo cp -i cockroach-latest.linux-amd64/cockroach` will not work because `cockroach-latest.linux-amd64` will not exist..

Reply:
That's really really difficult for me to make work with these cURL comands, because I have no way in the docs to know what version they will actually get. Let me play around with cURL for a minute.

Reply:
I worked around this by bringing Dedicated content in line with Serverless.

Reply:
Then Celia showed me that the Dedicated modal is way different. Latest commits fix the Dedicated content to match reality.